docs/example-session.md

```diff
@@ -29,28 +29,28 @@ Create an instance of 'Paratii' using default settings:
 
 We can now deploy the paratii contracts:
 
-    contracts = await paratii.deployAllContracts()
+    contracts = await paratii.getOrDeployContracts()
 
 
-At this point, all Paratii contracts will be available in `paratii.contracts`, under their class names.
+At this point, all Paratii contracts will be available in `contracts`, under their class names.
 
  A fresh  `ParatiiToken` contract will hav been deployed. Because you deployed it, you will have all the tokens:
 
-    await paratii.contracts.ParatiiToken.balanceOf(web3.eth.accounts[0])
+    await contracts.ParatiiToken.balanceOf(web3.eth.accounts[0])
```

Comment:
why? We just defined the contracts variable two lines before.

Reply:
Yes that's why I made it `contracts` instead of `paratii.contracts`. Maybe I'm not following you. Do you want me to put it back as `paratii.contracts` and update all instances of that in example-session.md?

Reply:
sorry, i invered the red and green :-) you are right, of course